src/cli/start.cpp

```diff
@@ -63,15 +65,16 @@ int start(const start_args& args,
     // find the current shell (zsh, bash, etc)
     auto shell = util::current_shell();
     if (!shell) {
-        fmt::print("[error] unable to determine the current shell because {}\n",
-                   shell.error());
+        spdlog::error("unable to determine the current shell because {}",
+                      shell.error());
+        // TODO: return?
```

Comment:
To do, here or in a separate PR. Presumably this should return something non-zero?

Reply:
Fixed to return 1.